boards/mro/cannode/canbootloader.cmake

```
@@ -0,0 +1,13 @@
+include (${CMAKE_CURRENT_LIST_DIR}/uavcan_board_identity)
+
+px4_add_board(
+	PLATFORM nuttx
+	VENDOR ark
```

```suggestion
	VENDOR mro
```

Update the model as well.